Psychtoolbox/PsychCal/MeasureDpi.m

```diff
@@ -1,6 +1,16 @@
-function dpi=MeasureDpi(theScreen)
-% dpi=MeasureDpi([theScreen])
-% Helps the user to accurately measure the screen's dots per inch.
+function [dpi, SP]=MeasureDpi(theScreen, CC)
+% dpi=MeasureDpi([theScreen], [CC]);
```

Comment:
The help text needs to read [dpi, SP]=MeasureDpi([theScreen], [CC]);
Iow. You need to note that not just dpi is returned, but also SPI.